doc/grahom.xml

```diff
@@ -798,3 +798,57 @@ false
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="MaximalCommonSubdigraph">
+  <ManSection>
+    <Oper Name="MaximalCommonSubdigraph" Arg="D1, D2"/>
+    <Returns> A digraph and two transformations </Returns>
+    <Description>
+  If <A>D1</A> and <A>D2</A> are digraphs without multiedges, then
+  <C>MaximalCommonSubdigraph</C> finds a maximal common subgraph <C>M</C> of
+  <A>D1</A>and <A>D2</A> with the maximum number of vertices. It then returns
+  <C>M</C> together with embeddings of <C>M</C> into <A>D1</A> and <A>D2</A>.
+
+<Example><![CDATA[
+gap> MaximalCommonSubdigraph(PetersenGraph(), CompleteDigraph(10));
+[ <immutable digraph with 2 vertices, 2 edges>, IdentityTransformation,
+  IdentityTransformation ]
+gap> MaximalCommonSubdigraph(PetersenGraph(),
+> DigraphSymmetricClosure(CycleDigraph(5)));
+[ <immutable digraph with 5 vertices, 10 edges>, IdentityTransformation,
+  IdentityTransformation ]
+gap> MaximalCommonSubdigraph(NullDigraph(0), CompleteDigraph(10));
+[ <immutable empty digraph with 0 vertices>, IdentityTransformation,
+  IdentityTransformation ]
+]]></Example>
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="MinimalCommonSuperdigraph">
+  <ManSection>
+    <Oper Name="MinimalCommonSuperdigraph" Arg="D1, D2"/>
+    <Returns> A digraph and two transformations </Returns>
+    <Description>
+  If <A>D1</A> and <A>D2</A> are digraphs without multiedges, then
+  <C>MinimalCommonSuperdigraph</C> finds a minimal common subgraph <C>M</C> of
+  <A>D1</A>and <A>D2</A> with the minimum number of vertices. It then returns
+  <C>M</C> together with embeddings of <A>D1</A> and <A>D2</A> into <C>M</C>.
```

Comment:
I think it would be a good idea to spell out the mathematical details about what a minimal common superdigraph is (presumably a digraph with the minimal possible number of vertices such that `D1` and `D2` both embed into it).

It would also be a good idea to explicitly say that it returns the answer as a list `[M, t1, t2]` where `t1` and `t2`...